packages/rescript-mui-material/src/components/InputBase.res

```diff
@@ -51,15 +51,43 @@ type color =
   | @as("warning") Warning
   | String(string)
 
+/**
+https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/enterkeyhint
+ */
+type enterKeyHint =
+  | @as("enter") Enter
+  | @as("done") Done
+  | @as("go") Go
+  | @as("next") Next
+  | @as("previous") Previous
+  | @as("search") Search
+  | @as("send") Send
+
+/**
+https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode
+ */
+type inputMode =
+  | @as("none") None
+  | @as("text") Text
+  | @as("decimal") Decimal
+  | @as("numeric") Numeric
+  | @as("tel") Tel
+  | @as("search") Search
+  | @as("email") Email
+  | @as("url") Url
+
 type inputBaseComponentProps = {
   ...CommonProps.clickableProps,
   name?: string,
   step?: int,
   min?: int,
   max?: int,
   @as("type") type_?: string,
+  @as("disableunderline") disableUnderline?: string,
```

Comment:
I removed `disableUnderline` in favor of adding it to the InputBase.props, was that wrong?
it was so that it also works with `<Select />`.

Also I don't know why this was type string here.

Reply:
Hm, must have been a merge issue on my end. I will address it soon

Reply:
@fhammerschmidt I've removed the offending line